.gitignore

```diff
@@ -75,4 +75,6 @@ steam_appid.txt
 xcode/Barony/Barony.xcodeproj/project.xcworkspace/*
 xcode/Barony/Barony.xcodeproj/xcuserdata/*
 *.ps1
-!LICENSE.nativefiledialog.txt
\ No newline at end of file
+!LICENSE.nativefiledialog.txt
+#Ignore Jetbrains IDE project directories
+/.idea
```

Comment:
Hope this isn't a bother! Just helpful to keep IDE stuff out of the project if anyone else uses a JetBrains IDE.